.gitignore

```diff
@@ -17,3 +17,12 @@ build/
 local.properties
 site/
 __pycache__
+iosApp/Podfile.lock
+demos/appyx-interactions/iosApp/Pods/*
```

Comment:
This won't scale very well if there are more modules. can you do

```
**/iosApp/Pods/*
**/iosApp/iosApp.xcworkspace/*
**/iosApp/iosApp.xcodeproj/*
!**/iosApp/iosApp.xcodeproj/project.pbxproj
```

(I've not tested this)